api/model/Organization.ts

```diff
@@ -1,13 +1,37 @@
 import { SchemaDefinition as def } from '@contember/schema-definition'
 import { Model } from '@contember/schema'
 import { Offer } from './Offer'
+import { District } from './District'
 
+export const OrganizationTypeEnum = def.createEnum(
+	'collegeInitiative',
+	'researchAndUniversitySector',
+	'governmentOrganization',
+	'privateOrganization',
+	'other',
+	'osvcPerson',
+	'municipality',
+	'nonprofit',
+	'foundation',
+	'media',
+	'church',
+	'volunteerInitiative'
+)
+
+// TODO: okres, kraj
 export class Organization {
 	name = def.stringColumn().notNull()
+	address = def.stringColumn().notNull()
+	identificationNumber = def.stringColumn().notNull()
+	website = def.stringColumn().notNull()
```

Comment:
Tato selže migrace, protože to bude `null` pro existující. Buď to tedy dát jako nullable (není potřeba explicitně uvádět) nebo přidat výchozí hodnotu (`.default("...")`)

Reply:
Ok, diky za heads-up